.ci/targets/android_platform_tests.yaml

```diff
@@ -7,6 +7,10 @@ tasks:
   - name: lint
     script: script/tool_runner.sh
     args: ["lint-android"]
+  - name: download android deps
```

Comment:
Based on conversations from https://flutter-review.googlesource.com/c/recipes/+/46980, it is recommended to move the infra step to the very beginning.